contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM resin/rpi-raspbian:jessie
```

Comment:
I agree, but we probably need an ARMv6 compatible base image here. The armhf/debian:jessie is for ARMv7. I'm not sure if the `ENV GOARM 6` below is enough to have proper docker engine binaries for ARMv6.